scripts/generate_proto_stubs.sh

```diff
@@ -11,7 +11,7 @@ set -ex -o pipefail
 # followed by committing the changes to typeshed
 #
 # Update these two variables when rerunning script
-PROTOBUF_VERSION=3.19.3
+PROTOBUF_VERSION=3.20.1
 MYPY_PROTOBUF_VERSION=v3.2.0
```

Comment:
```suggestion
MYPY_PROTOBUF_VERSION=v3.3.0
```

Reply:
yes, but I'd like to do it separately - because it ends up having a pretty different set of changes.

I originally was preparing a PR that did both all at once, but it ends up with a bunch of different changes since it changes the code-generator utility.

Would prefer to keep the two PRs isolated.

Reply:
Sure, makes sense!

Reply:
I was also planning to test the changes with 3.3.0 and if things don't work out perfectly, I may have to release 3.3.1 (or 3.4.0) to get typeshed to work.